native-sql-engine/core/src/main/scala/com/intel/oap/execution/ArrowRowToColumnarExec.scala

```diff
@@ -65,7 +65,7 @@ class ArrowRowToColumnarExec(child: SparkPlan) extends RowToColumnarExec(child =
         case d: BinaryType =>
```

Comment:
We've supported ArrayType (not supported nested type) for ArrowRowToColumnarExec but not add it on "buildCheck". We can pass the unit tests by adding the code like following picture:

![image](https://user-images.githubusercontent.com/44394375/149892952-b9aa1e42-94e2-4f55-a14d-22453ee4c8c5.png)